### PR TITLE
Introduce skeleton pods service and api 🎉 

### PR DESCRIPTION
**Pull request by krisnova**
Signed-off-by: Kris Nóva <kris@nivenly.com>

### CODE REVIEW

File: api/v0/runtime.proto [hunk 1]
```diff
@@ -68,6 +65,31 @@ service Spawn {}
...
+/// will look at feel like one or more "containers".
+///
+/// Pods will run an OCI compliant container image.
+///
+/// A pod is a group of one or more containers with shared network and storage.
+service PodService {
+
+  rpc Allocate(AllocatePodRequest) returns (AllocatePodResponse) {}
+  rpc Start(StartContainerRequest) returns (StartContainerResponse) {}
+  rpc Stop(StopContainerRequest) returns (StopContainerResponse) {}
```
Comment (future-highway) [inline api/v0/runtime.proto:78]: I still prefer these to be more verbose in their naming. E.g., `AllocatePod`, `StartContainer`, `StopContainer`, `FreePod`.

Not a big deal though

**Review by future-highway — CHANGES_REQUESTED** [review]
Blocking the merge because the macro is not generating the right deno code.

aka, I'm requesting changes from myself

File: auraescript/src/runtime.rs [hunk 2]
```diff
@@ -0,0 +1,49 @@
+/* -------------------------------------------------------------------------- *\
```
Comment (dmah42) [inline auraescript/src/runtime.rs:1]: is this a generated file now?  it seems weird to have to repeat the contents of the proto.

Comment (dmah42) [reply auraescript/src/runtime.rs:1]: oh i guess we did that anyway.

**Review by dmah42 — DISMISSED** [review]

**Review by future-highway — APPROVED** [review]
LGTM

**Pull request merged**